pages/common/gist.md

```diff
@@ -3,7 +3,7 @@
 > Upload code to https://gist.github.com.
 > More information: <https://github.com/defunkt/gist>.
 
-- Log in in gist on this computer:
+- Log in gist on this computer:
```

Comment:
```suggestion
- Log in to gist on this computer:
```
The second "in" was being used in the sense of "to", so I think it's better to use "to"